### ISSUE TITLE
UptoBox / XFileSharing plugins

### ISSUE THREAD

**Issue by STarfield42** [pyload/pyload#3293]
Hi,

just installed a fresh pyload on a Raspberry Pi/Raspbian Stretch Lite.
Works fine with simple URL : BasePlugin does the job.

With premium account on UpToBox.com, I get errors as described with issue #3140.
- If XFileSharing is active, I get the "TEXTAREA F1 not found" error like moacmoa
- if XFileSharing is not active, I get the "'NoneType' object has no attribute 'group'" like xilmen.

I try to get this URL : https://uptobox.com/md2kfd8nmqa3

Here is the log:
```
09.12.2018 19:27:00 CRITICAL  'NoneType' object has no attribute 'group'
Traceback (most recent call last):
  File "/usr/share/pyload/module/ThreadManager.py", line 276, in assignJob
    job.initPlugin()
  File "/usr/share/pyload/module/utils.py", line 165, in new
    return func(*args)
  File "/usr/share/pyload/module/PyFile.py", line 103, in initPlugin
    self.plugin = self.pluginclass(self)
  File "/root/.pyload/userplugins/internal/Base.py", line 99, in __init__
    self.init()
  File "/root/.pyload/userplugins/hoster/XFileSharing.py", line 39, in init
    self.pyfile.url).group("DOMAIN").lower()
AttributeError: 'NoneType' object has no attribute 'group'
```
If I delete my premium account data, I get the same error.

**Comment by GammaC0de** [comment]
@STarfield42 I tested the above link and it worked fine with premium and free download, XFileSharing disabled.

How exactly did you installed pyload?

**Comment by GammaC0de** [comment]
Are you running pyload as user "pyload"?

Does the "pyload" user has write access to /opt/pyload directory?

**Comment by STarfield42** [comment]
Yes, pyload is running with the pyload user (checked with top command)

But you right, the "/opt/pyload" and the "/home/pyload/" folders were not owned by pyload (root instead) !
So I did this command "chown -R pyload:nogroup /opt/pyload" and I deleted the /home/pyload/.pyload folder.

After a new setup, it seems to be better : it works with free account.

At this time I can't add my premium account: "Login handshake has failed".
It's late in France so I'll do some test tomorrow.

Thank you so much @GammaC0de for your help.
